ledger/partial/ptrie/partialTrie_test.go

```diff
@@ -115,7 +114,7 @@ func TestPartialTrieLeafUpdates(t *testing.T) {
 		_, err = psmt.Update(paths, payloads)
 		require.NoError(t, err, "error updating psmt")
 
-		if !bytes.Equal(rootHash, psmt.root.HashValue()) {
+		if hash.Hash(rootHash) != psmt.root.HashValue() {
```

Comment:
just to want to double check that go's behaviour on this inequality check is not the address pointer and value itself ?

Reply:
Yep, Go treats arrays as values (like `int`, there are no pointers), so the comparison does work on the actual values.